typings/index.d.ts

```diff
@@ -1501,6 +1507,17 @@ declare module 'discord.js' {
 		reason?: string;
 	};
 
+	type BanInfo = {
+		user: User;
+		fetched: boolean;
+		reason?: string;
```

Comment:
I think this should be `reason: string | null` instead.
Currently this means undefined / not present.
It is always present however, just may be null.